modules/imap/output_modules.php

```diff
@@ -1383,3 +1392,24 @@ protected function output() {
         return $res;
     }
 }
+
+/**
+ * Option to set the per page count for IMAP folder views
+ * @subpackage imap/output
+ */
+class Hm_Output_first_time_screen_emails_per_page_setting extends Hm_Output_Module {
+    protected function output() {
+        $settings = $this->get('user_settings', array());
+        $per_page = 20;
+        $reset = '';
+        if (array_key_exists('first_time_screen_emails', $settings)) {
```

Comment:
I'm getting this by default:
<img width="1211" alt="Screenshot 2024-07-31 at 17 23 48" src="https://github.com/user-attachments/assets/4389b36a-c5dd-46fa-917e-e1a432ed5f9b">

rewrite the condition like this:
```
if (array_key_exists('first_time_screen_emails', $settings) && $settings['first_time_screen_emails']) {
            $per_page = $settings['first_time_screen_emails'];
}
```
to resolve and make it work correctly.

You will just add `&& $settings['first_time_screen_emails']` in your condition.

Reply:
@christer77 Will you have time to fix minor issues ?

Reply:
Noted